app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+const { getTopics } = require('./controllers/app.controllers');
+
+app.get('/api/topics', getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    console.log(err);
+    res.status(500).send({ msg: 'Internal Server Error' });
+  }
+});
```

Comment:
Its nice to see error handling added but this should not include custom error handling at this stage as we have no way of confirming this is working as expected. The 500 server error is a nice fallback to have and the only one needed at this stage.